generators/entity-server/templates/src/main/kotlin/package/domain/Entity.kt.ejs

```diff
@@ -208,7 +208,7 @@ import org.springframework.data.couchbase.core.mapping.id.GenerationStrategy.UNI
 <%_ } if (searchEngine === 'elasticsearch') { _%>
 @org.springframework.data.elasticsearch.annotations.Document(indexName = "<%= entityInstance.toLowerCase() %>")
 <%_ } _%>
-class <%= asEntity(entityClass) %>(
+data class <%= asEntity(entityClass) %>(
```

Comment:
I am sorry I haven't replied back earlier. This is an exciting change, but initially I had problems with this that is the reason why I reverted back to normal class implementation. 

The builds are failing too here. Can we fix the build?